spec/client_configuration_spec.rb

```diff
@@ -1,5 +1,11 @@
 require 'spec_helper'
 
+def os_info
```

Comment:
I'd rather recreate it just for the sake of simplicity, I can add a spec to check that this function does what's expected, but as it depends on each platform, I cannot assure that the same output will be valid in any machine that's running the unit tests.